helmfile/values/harbor-certs.yaml.gotmpl

```
@@ -1,5 +1,5 @@
 core:
-  domain: harbor.{{ .Values.global.baseDomain }}
+  domain: {{ .Values.harbor.subdomain }}.{{ .Values.global.baseDomain }}
 
 notary:
-  domain: notary.harbor.{{ .Values.global.baseDomain }}
+  domain: {{ .Values.harbor.notary.subdomain }}.{{ .Values.harbor.subdomain }}.{{ .Values.global.baseDomain }}
```

I think it would make sense to merge these two so this would be
```suggestion
  domain: {{ .Values.harbor.notary.subdomain }}.{{ .Values.global.baseDomain }}
```

And then the default config for notary would be:

```yaml
harbor:
  notary:
    subdomain: notary.harbor
```

Then you could change this to not include harbor if you really wanted.